src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -1,25 +1,24 @@
 @use 'sass:map';
 @use '../../material/core/style/layout-common';
+@use '../../material/core/style/focus-indicators';
 
 /// Mixin that turns on strong focus indicators.
 ///
 /// @example
 ///   .my-app {
 ///     @include mat-mdc-strong-focus-indicators($config);
 ///   }
+
 @mixin strong-focus-indicators($config: ()) {
   // Default focus indicator config.
   $default-config: (
-    border-style: solid,
-    border-width: 3px,
-    border-radius: 4px,
+    border-color: black,
   );
 
   // Merge default config with user config.
   $config: map.merge($default-config, $config);
-  $border-style: map.get($config, border-style);
-  $border-width: map.get($config, border-width);
-  $border-radius: map.get($config, border-radius);
+
+  @include focus-indicators.private-customize-focus-indicators($config, 'mat-mdc');
 
   // Base styles for focus indicators.
   .mat-mdc-focus-indicator::before {
```

Comment:
I may be missing something, but aren't these structural styles are already included in `_core.scss`? Can't we remove these from here and also the legacy `_focus-indicators.scss`? My understanding was that the `strong-focus-indicator` mixin literally just defines CSS variables, that's it.